pkgs/applications/networking/irc/weechat/scripts/colorize_nicks/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "weechat-colorize_nicks";
+  version = "27";
+
+  src = fetchurl {
+    url = "https://github.com/weechat/scripts/raw/bc8a9051800779a036ba11689a277cd5f03657b2/python/colorize_nicks.py";
+    sha256 = "0hiay88vvy171jiq6ahflm0ipb7sslfxwhmmm8psv6qk19rv2sxs";
+  };
+
+  unpackPhase = ''
+    :
+  '';
```

Comment:
```suggestion
  doUnpack = true;
```

Reply:
I think you mean `false` here, since `:` is a noop.

Reply:
Do you have a reference for `doUnpack`? I don't seem to be able to find it anywhere.

Reply:
It’s `dontUnpack`